lib/traject/macros/xml.rb

```diff
@@ -21,6 +21,11 @@ def apply_translation(values, options)
       translation_map = Traject::TranslationMap.new(options[:translation_map])
       # without overwriting (further) translation map, could add
       # fuzzy match method here after pulling array out of TM
+      if values.is_a? Array
+        values = values.map(&:downcase)
+      elsif values.is_a? String
+        values = value.downcase
+      end
       translation_map.translate_array! values
```

Comment:
Can the value be a string? I thought all these traject things were arrays.

(Also, `translate_array!` below can/should probably be `translate_array`? )